src/Facets/DeBridgeDlnFacet.sol

```diff
@@ -190,6 +191,10 @@ contract DeBridgeDlnFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
                 allowedTakerDst: "",
                 externalCall: "",
                 allowedCancelBeneficiarySrc: abi.encodePacked(msg.sender)
+                // NOTE: The allowedCancelBeneficiarySrc is intentionally set to msg.sender,
+                // which, when called via Permit2Proxy, becomes the proxy contract address.
+                // This ensures that if a DLN order is cancelled, funds are refunded to Permit2Proxy.
+                // The user can later withdraw them via Permit2Proxy's WithdrawablePeriphery interface.
```

Comment:
This comment is not entirely correct. A user cannot withdraw tokens using the WithdrawablePeriphery functionality as this is an admin-only function. Please could you update the comment?